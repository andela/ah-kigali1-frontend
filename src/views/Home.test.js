describe("test dummy test", () => {
  test("it test dummy test", () => {
    expect(true).toBe(true);
  });
});
