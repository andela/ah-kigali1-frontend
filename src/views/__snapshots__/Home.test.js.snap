// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component it matches the snapshot 1`] = `
<div>
  <h2
    className="home-container"
  >
    Hello world, from Titan-Devs
  </h2>
</div>
`;
